project/OmnidocBuild.scala

```diff
@@ -195,12 +180,14 @@ object OmnidocBuild {
 
   def compilerReporterSettings = Seq(
     compile / compilerReporter := {
-      new sbt.internal.server.LanguageServerReporter(
+      new sbt.internal.inc.LoggedReporter(
```

Comment:
This and the below tastyFiles / bspReporter hack allows us to finally upgrade to latest sbt version